src/L2ERC20.sol

```diff
@@ -5,18 +5,25 @@ import {ERC20} from "openzeppelin/token/ERC20/ERC20.sol";
 import {ERC20Votes} from "openzeppelin/token/ERC20/extensions/ERC20Votes.sol";
 import {ERC20Permit} from "openzeppelin/token/ERC20/extensions/ERC20Permit.sol";
 import {Ownable} from "openzeppelin/access/Ownable.sol";
+import {SafeCast} from "openzeppelin/utils/math/SafeCast.sol";
 import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
 import {WormholeReceiver} from "src/WormholeReceiver.sol";
+import {IL1Block} from "src/interfaces/IL1Block.sol";
 
 contract L2ERC20 is ERC20Votes, WormholeReceiver {
+  /// @notice The contract that handles fetch the L1 block on the L2.
```

Comment:
nit: "fetching" not "fetch"

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/9/commits/a94254ea995a0af202c4cc1005019ea7866482b9